app/exceptions/city_exc.py

```diff
@@ -24,13 +24,36 @@ def __init__(
         self.status_code = status_code
 
 
-class ZipCodeNotFoundError(Exception):
+class CityOutOfRangeError(Exception):
     def __init__(
         self,
+        expected_type: dict = {},
```

Comment:
This should be something like "cities_in_our_database_range", or something like this.

Reply:
Done.